knack/index.css

```diff
@@ -60,23 +46,45 @@ div.view_2465 .field_3174 {
 }
 
 .big-button-container {
-  border-radius: 2px;
-  box-shadow: 0px 1px 2px 0px gray;
-  font-size: 2.5em;
-  padding: 10px;
-  margin: 20px;
-  max-width: 16em;
-}
-
+    border-radius: 2px;
+    box-shadow: 0px 1px 2px 0px gray;
+    font-size: 2.5em;
+    padding: 10px;
+    margin: 20px;
+    max-width: 16em;
+}
+ 
 .big-button-container:hover {
-  background-color: #f7f7f7;
-  cursor: pointer;
+    background-color: #f7f7f7;
+    cursor: pointer;
 }
-
+ 
+ 
 .fa {
-  vertical-align: middle;
+vertical-align: middle;
 }
-
+ 
+ 
 a.big-button {
   text-decoration: none;
 }
+
+ a.small-button {
+  text-decoration: none;
+}
+
+.small-button-container {
+  padding: 5px;
+  margin: 20px;
+  border-radius: 2px;
+  box-shadow: 0px 1px 2px 0px gray;
+  font-size: 1em;
+  max-width: 15em;
+  background-color: #828282;
+  color: white;
+}
+
+.small-button-container:hover {
+    background-color: #4c4c4c;
```

Comment:
Every time CSS is inconsistently indented a kitten dies.

Reply:
😹